config/components.yml

```diff
@@ -57,3 +57,4 @@ frameworks:
   - "JavaBuildpack::Framework::SpringInsight"
   - "JavaBuildpack::Framework::YourKitProfiler"
   - "JavaBuildpack::Framework::JavaOpts"
+  - "JavaBuildpack::Framework::FusionReactor"
```

Comment:
This should go into the list alphabetically.  `JavaOpts` is a special case to ensure it overrides everything else.